### PR TITLE
IGNITE-17995 Idle_verify must print broken partitions list

### PR DESCRIPTION
**Pull request by shishkovilja**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [x] There is a single JIRA ticket related to the pull request. 
- [x] The web-link to the pull request is attached to the JIRA ticket.
- [x] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [x] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [x] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [x] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java [hunk 1]
```diff
@@ -514,13 +520,19 @@ public void testCacheIdleVerifyTwoConflictTypes() throws Exception {
 
         GridCacheContext<Object, Object> cacheCtx = ignite.cachex(DEFAULT_CACHE_NAME).context();
 
-        corruptDataEntry(cacheCtx, 1, true, false);
+        AffinityFunction aff = cacheCtx.config().getAffinity();
```
Comment (anton-vinogradov) [inline modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java:523]: newline missed

File: modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java [hunk 2]
```diff
@@ -851,9 +863,11 @@ private void corruptingAndCheckDefaultCache(
 
         GridCacheContext<Object, Object> cacheCtx = ignite.cachex(DEFAULT_CACHE_NAME).context();
 
-        corruptDataEntry(cacheCtx, 0, true, false);
+        AffinityFunction aff = cacheCtx.config().getAffinity();
```
Comment (anton-vinogradov) [inline modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java:866]: newline missed

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/IdleVerifyResultV2.java [hunk 3]
```diff
@@ -266,31 +272,59 @@ private void printConflicts(Consumer<String> printer) {
...
+        printer.accept("Total:" + nl());
+
+        Map<String, TreeSet<Integer>> conflictsSummary = allConflicts.stream()
+            .collect(Collectors.groupingBy(
+                PartitionKeyV2::groupName,
+                Collectors.mapping(
+                    PartitionKeyV2::partitionId,
+                    Collectors.toCollection(TreeSet::new))));
+
+        for (String groupName : conflictsSummary.keySet()) {
```
Comment (anton-vinogradov) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/IdleVerifyResultV2.java:315]: Let's iterate using the Map.Entry

**Review by anton-vinogradov — CHANGES_REQUESTED** [review]

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/IdleVerifyResultV2.java (hunk 3 above)
Comment (shishkovilja) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/IdleVerifyResultV2.java:315]: Good point. I'll prepare a fix soon.

File: modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java (hunk 2 above)
Comment (shishkovilja) [reply modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java:872]: Good point. I'll prepare a fix soon.

File: modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java (hunk 1 above)
Comment (shishkovilja) [reply modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java:528]: Good point. I'll prepare a fix soon.

**Review by anton-vinogradov — APPROVED** [review]

**Pull request merged**